app/views/api/v0/articles/index.json.jbuilder

```diff
@@ -1,5 +1,5 @@
 json.array! @articles do |article|
-  json.partial! "article", article: article
+  json.partial! "api/v0/shared/article", article: article
```

Comment:
❤️

Reply:
I changed it back since I deleted the articles jbuilder file